File: Pattern 5/Pattern_5.cpp
class Solution{
public:
	
	void printTriangle(int n) {
	    // iterate on the loop in reverse direction
	    // for every i print i asterisk(*)
	    for(int i =n; i>0; i--) {
	        for(int j =1; j<=i; j++) {
	            cout<<"* ";
	        }
	        cout<<endl;
	    }
	}
};

File: Pattern 5/Pattern_5.py

class Solution:
    def printTriangle(self, n):
        # iterate on the loop in reverse direction
	    # for every i print i asterisk(*)
	    for i in range(n,0,-1):
	        for j in range(1,i+1):
	            print("*",end= " ")
	        print()
	       